docs/user-guide/jobs.md

```diff
@@ -0,0 +1,256 @@
+```
+
+## Writing a Job Spec
+
+As with all other Kubernetes config, a Job needs `apiVersion`, `kind`, and `metadata` fields.  For
+general information about working with config files, see [here](simple-yaml.md),
+[here](configuring-containers.md), and [here](working-with-resources.md).
+
+A Job also needs a [`.spec` section](../devel/api-conventions.md#spec-and-status).
+
+### Pod Template
+
+The `.spec.template` is the only required field of the `.spec`.
+
+The `.spec.template` is a [pod template](replication-controller.md#pod-template).
+ It has exactly the same schema as a [pod](pods.md), except
+it is nested and does not have an `apiVersion` or `kind`.
+
+In addition to required fields for a Pod, a pod template in a job must specify appropriate
+lables (see [pod selector](#pod-selector) and an appropriate restart policy.
+
+Only a [`RestartPolicy`](pod-states.md) equal to `Never` or `OnFailure` are allowed.
+
+### Pod Selector
+
+The `.spec.selector` field is a pod selector.  It works the same as the `.spec.selector` of
+a [ReplicationController](replication-controller.md).
+
+If specified, the `.spec.template.metadata.labels` must be equal to the `.spec.selector`.  Config not following
+this rule will be rejected by the API.  If unspecified, it will be defaulted to the latter.
```

Comment:
I think the 'latter' should rather be the former in here, but it'll be best, to replace it with straight information that it takes value from `.spec.template.metadata.labels`.